BUILD.bazel

```diff
@@ -0,0 +1,23 @@
+# Copyright 2018 Google LLC
```

Comment:
We probably want to add some documentation to the bazel files :) Major docs

Reply:
I will provide full documentation on both repo/demo flow/development flow and also walkthrough of setting up bazel like this. 

But, if it's ok, I'd like to finish moving things around first and get the tests going ( issues #3 and #4 ). I think taking care of those issues will help to focus the docs.